internal/transactionsubmission/engine/signing/distribution_account_resolver.go

```diff
@@ -25,13 +26,18 @@ type DistributionAccountResolver interface {
 type DistributionAccountResolverOptions struct {
 	AdminDBConnectionPool            db.DBConnectionPool
 	HostDistributionAccountPublicKey string
+	MTNDBConnectionPool              db.DBConnectionPool
 }
 
 func (c DistributionAccountResolverOptions) Validate() error {
 	if c.AdminDBConnectionPool == nil {
 		return fmt.Errorf("AdminDBConnectionPool cannot be nil")
 	}
 
+	if c.MTNDBConnectionPool == nil {
+		return fmt.Errorf("MTNDBConnectionPool cannot be nil")
+	}
+
```

Comment:
🤔  This could be made optional for the resolver, since it's not needed in the channel accounts, nor TSS stuff. WDYT?

👋  If you prefer to keep it mandatory, plz remember to update the code in cmd/transaction_submitter.go as well. Right now, the tss command is broken due to this change.

Reply:
Addressed in https://github.com/stellar/stellar-disbursement-platform-backend/pull/328/commits/6f25d63653868f201f1c857296ae46b48beeb100